### DISCUSSION TITLE
element-plus源码中 /package/components/xxx/style文件 用处

### DISCUSSION THREAD

**Discussion by itshelena** [element-plus/element-plus#11927] [Q&A]
element-plus源码中主题是单独打包的，且在打包样式文件时 并未涉及到 /package/components 文件中具体组件中 style 文件，这个style文件是否多余呢
<img width="192" alt="image" src="https://user-images.githubusercontent.com/25857241/223911423-b338bdfb-909c-4268-9840-dc391bccca64.png">

**Comment by michael20102013** [comment]
按需引入的时候需要用到
